Makefile.am

```diff
@@ -4356,6 +4356,26 @@ dist_catalog_DATA = \
 	catalog/systemd.zh_TW.catalog \
 	catalog/systemd.catalog
 
+EXTRA_DIST += \
+	catalog/systemd.bg.catalog.in \
+	catalog/systemd.be.catalog.in \
+	catalog/systemd.be@latin.catalog.in \
+	catalog/systemd.fr.catalog.in \
+	catalog/systemd.it.catalog.in \
+	catalog/systemd.pl.catalog.in \
+	catalog/systemd.pt_BR.catalog.in \
+	catalog/systemd.ru.catalog.in \
+	catalog/systemd.zh_CN.catalog.in \
+	catalog/systemd.zh_TW.catalog.in \
+	catalog/systemd.catalog.in
+
```

Comment:
Maye use something like
```
 EXTRA_DIST += \
-       catalog/systemd.bg.catalog.in \
-       catalog/systemd.be.catalog.in \
-       catalog/systemd.be@latin.catalog.in \
-       catalog/systemd.fr.catalog.in \
-       catalog/systemd.it.catalog.in \
-       catalog/systemd.pl.catalog.in \
-       catalog/systemd.pt_BR.catalog.in \
-       catalog/systemd.ru.catalog.in \
-       catalog/systemd.zh_CN.catalog.in \
-       catalog/systemd.zh_TW.catalog.in \
-       catalog/systemd.catalog.in
+       $(nodist_catalog_DATA:.catalog=.catalog.in)
```
instead. This would avoid that the .catalog and .catalog.in lists get out of sync